### ISSUE TITLE
[v2] wails build of unmodified template fails

### ISSUE THREAD

**Issue by TechplexEngineer** [wailsapp/wails#731]
**Description**
Created sample/template project, unable to build into binary. Dev mode works great

**To Reproduce**
Steps to reproduce the behaviour:
1. `wails init -n test`
2. `cd test/frontend`
3. `npm install`
4. `wails build`

Full error output:
```
wails build
Wails CLI v2.0.0-alpha.65


App Type: 		desktop
Platform: 		linux
Arch: 			amd64
Compiler: 		go
Compress: 		false
Build Mode: 		debug
Package: 		false
Clean Build Dir: 	false
KeepAssets: 		false
LDFlags: 		""
Tags: 			[]

Installing frontend dependencies: Done.
Compiling frontend: Done.
Building assets: Done.
Compiling application: 

ERROR: exit status 2
# github.com/wailsapp/wails/v2/internal/ffenestri
ffenestri_linux.c: In function ‘DarkModeEnabled’:
ffenestri_linux.c:856:22: error: parameter name omitted
  856 | void DarkModeEnabled(struct Application*, char *callbackID) {}
      |                      ^~~~~~~~~~~~~~~~~~~
ffenestri_linux.c: In function ‘SetApplicationMenu’:
ffenestri_linux.c:857:25: error: parameter name omitted
  857 | void SetApplicationMenu(struct Application*, const char *) {}
      |                         ^~~~~~~~~~~~~~~~~~~
ffenestri_linux.c:857:46: error: parameter name omitted
  857 | void SetApplicationMenu(struct Application*, const char *) {}
      |                                              ^~~~~~~~~~~~
ffenestri_linux.c: In function ‘AddTrayMenu’:
ffenestri_linux.c:858:18: error: parameter name omitted
  858 | void AddTrayMenu(struct Application*, const char *menuTrayJSON) {}
      |                  ^~~~~~~~~~~~~~~~~~~
ffenestri_linux.c: In function ‘SetTrayMenu’:
ffenestri_linux.c:859:18: error: parameter name omitted
  859 | void SetTrayMenu(struct Application*, const char *menuTrayJSON) {}
      |                  ^~~~~~~~~~~~~~~~~~~
ffenestri_linux.c: In function ‘DeleteTrayMenuByID’:
ffenestri_linux.c:860:25: error: parameter name omitted
  860 | void DeleteTrayMenuByID(struct Application*, const char *id) {}
      |                         ^~~~~~~~~~~~~~~~~~~
ffenestri_linux.c: In function ‘UpdateTrayMenuLabel’:
ffenestri_linux.c:861:26: error: parameter name omitted
  861 | void UpdateTrayMenuLabel(struct Application*, const char* JSON) {}
      |                          ^~~~~~~~~~~~~~~~~~~
ffenestri_linux.c: In function ‘AddContextMenu’:
ffenestri_linux.c:862:21: error: parameter name omitted
  862 | void AddContextMenu(struct Application*, char *contextMenuJSON) {}
      |                     ^~~~~~~~~~~~~~~~~~~
ffenestri_linux.c: In function ‘UpdateContextMenu’:
ffenestri_linux.c:863:24: error: parameter name omitted
  863 | void UpdateContextMenu(struct Application*, char *contextMenuJSON) {}
      |                        ^~~~~~~~~~~~~~~~~~~
ffenestri_linux.c: In function ‘WebviewIsTransparent’:
ffenestri_linux.c:864:27: error: parameter name omitted
  864 | void WebviewIsTransparent(struct Application*) {}
      |                           ^~~~~~~~~~~~~~~~~~~
ffenestri_linux.c: In function ‘WindowBackgroundIsTranslucent’:
ffenestri_linux.c:865:36: error: parameter name omitted
  865 | void WindowBackgroundIsTranslucent(struct Application*) {}
      |                                    ^~~~~~~~~~~~~~~~~~~
ffenestri_linux.c: In function ‘OpenDialog’:
ffenestri_linux.c:866:17: error: parameter name omitted
  866 | void OpenDialog(struct Application*, char *callbackID, char *title, char *filters, char *defaultFilename, char *defaultDir, int allowFiles, int allowDirs, int allowMultiple, int showHiddenFiles, int canCreateDirectories, int resolvesAliases, int treatPackagesAsDirectories) {}
      |                 ^~~~~~~~~~~~~~~~~~~
ffenestri_linux.c: In function ‘SaveDialog’:
ffenestri_linux.c:867:17: error: parameter name omitted
  867 | void SaveDialog(struct Application*, char *callbackID, char *title, char *filters, char *defaultFilename, char *defaultDir, int showHiddenFiles, int canCreateDirectories, int treatPackagesAsDirectories) {}
      |                 ^~~~~~~~~~~~~~~~~~~
ffenestri_linux.c: In function ‘MessageDialog’:
ffenestri_linux.c:868:20: error: parameter name omitted
  868 | void MessageDialog(struct Application*, char *callbackID, char *type, char *title, char *message, char *icon, char *button1, char *button2, char *button3, char *button4, char *defaultButton, char *cancelButton) {}
      |                    ^~~~~~~~~~~~~~~~~~~

```

**Expected behaviour**
A statically linked binary to be produced.

**Screenshots**
Works in dev mode
![image](https://user-images.githubusercontent.com/175873/120856872-6fdff000-c54e-11eb-9c6e-e52f968f71eb.png)

**System Details**
Linux Mint 20 Ulyana (based on Ubuntu 20.04)
go version go1.16.2 linux/amd64

**Comment by leaanthony** [comment]
Thanks for opening this. I can see why this would fail, yet it built on my VM last night. Will fix today 👍

**Comment by leaanthony** [comment]
This should be fixed in `v2.0.0-alpha.66`. Please let me know how you get on 👍

**Comment by leaanthony** [comment]
Fairly certain this is fixed now
